### DISCUSSION TITLE
Uncaught exception occured.

### DISCUSSION THREAD

**Discussion by Cimedaca618** [Monika-After-Story/MonikaModDev#9349] [QA - tech support + general] [answered]
Greetings. I've updated MAS and got myself this.
Can you help me?
```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/zz_spriteobjects.rpy", line 622, in script
  File "game/zz_spriteobjects.rpy", line 788, in <module>
AttributeError: 'StoreModule' object has no attribute 'EXP_C_WET'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/zz_spriteobjects.rpy", line 622, in script
  File "C:\DDLC\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\DDLC\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/zz_spriteobjects.rpy", line 788, in <module>
AttributeError: 'StoreModule' object has no attribute 'EXP_C_WET'

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.9

```

**Comment by Booplicate** [comment]
Show your `game/` folder.

**Comment by Cimedaca618** [comment]
> .
Granted!
![изображение](https://user-images.githubusercontent.com/36435169/180324933-26fcc298-003f-4cd5-a4a7-964e730a2583.png)

**Comment by Booplicate** [comment]
At the top open the `View` tab, then in `layout` select `Details` so we can see the dates on the files.

**Comment by Cimedaca618** [comment]
!
I accidentally closed it. My bad.
![изображение](https://user-images.githubusercontent.com/36435169/180325965-82325db7-33c7-40cb-9990-e281994208c9.png)

**Comment by Booplicate** [comment, marked as answer]
Your `sprite-chart.rpyc` has a later date, you replaced it with an older version and got a crash. Reinstall the mod into the same directory to fix that.

**Comment by Cimedaca618** [comment]
Alright, give me a second to try your solution...

**Comment by Cimedaca618** [comment]
Aaand done! Everything works, thank you!
